### ISSUE TITLE
I translated the client and uploaded the new one.  The name above the avatar. I can't see it.  How to solve it?  

### ISSUE THREAD

**Issue by justinzm** [mozilla/hubs#5284]
I translated the client and uploaded the new one.  The name above the avatar. I can't see it.  How to solve it?  

![122](https://user-images.githubusercontent.com/6271369/159615028-d377830d-e119-48d2-b257-bfa699c4fadc.png)
![321](https://user-images.githubusercontent.com/6271369/159615090-bf9743f8-5e6a-4494-beb0-2d99ef13c247.jpg)

**Comment by matthewbcool** [comment]
I guess this is happening on hubs cloud and you hardcoded translations? Or is this hubs.mozilla.com? 

Screenshots are helpful but please include more details such as the platform/browser, a clearer description and steps to reproduce as requested earlier.

**Comment by justinzm** [comment]
Yes , we are using AWS cloud and translated by hardcoding.platform PC OS Win10 google-chrome -v: 98.0.4758.102

**Comment by justinzm** [comment]
> Yes , we are using AWS cloud and translated by hardcoding.platform PC OS Win10 google-chrome -v: 98.0.4758.102

Yes , we are using AWS cloud and translated by hardcoding.platform PC OS Win10 google-chrome -v: 98.0.4758.102

**Comment by justinzm** [comment]
> I guess this is happening on hubs cloud and you hardcoded translations? Or is this hubs.mozilla.com?
> 
> Screenshots are helpful but please include more details such as the platform/browser, a clearer description and steps to reproduce as requested earlier.

Yes , we are using AWS cloud and translated by hardcoding.platform PC OS Win10 google-chrome -v: 98.0.4758.102
